core/commands/object/object.go

```diff
@@ -24,6 +25,12 @@ type Node struct {
 	Data  string
 }
 
+type NodeWithMeta struct {
+	Links    []Link
+	Data     string
```

Comment:
You can put it like this

```go
type NodeWithMeta struct {
    Node

    Metadata string
}
```

Reply:
Should `Metadata` be a `map[string]interface{}`? We talked about unmarshalling it for better display (no quotes)?